lib/tasks/test.rake

```diff
@@ -167,6 +167,17 @@ namespace :test do
     end
   end
 
+  desc 'abort execution if elasticsearch is not running'
+  task :abort_if_elasticsearch_is_not_running do
+    elasticsearch_port = ENV['ELASTIC_TEST_PORT'].present?
+    elasticsearch_port = '9250' unless elasticsearch_port.present?
+    unless port_open?('127.0.0.1', elasticsearch_port)
```

Comment:
if `ENV['ELASTIC_TEST_PORT']` is actually `present?` the value of `elasticsearch_port` will be `true`, which is probably not what you want.